containers/buildinfo.go

```diff
@@ -57,15 +66,37 @@ func GetBuildInfo(ctx context.Context, d *dagger.Client, dir *dagger.Directory,
 	timestamp := time.Now()
 
 	result := &BuildInfo{
-		Version:   version,
-		Commit:    sha,
-		Branch:    branch,
-		Timestamp: timestamp,
+		Version:          version,
+		Commit:           sha,
+		EnterpriseCommit: enterpriseSha,
+		Branch:           branch,
+		Timestamp:        timestamp,
 	}
 	span.SetAttributes(attribute.String("version", version), attribute.String("commit", sha), attribute.String("branch", branch), attribute.String("timestamp", timestamp.Format(time.RFC3339)))
```

Comment:
I'm not sure about the goal of this `span.SetAttributes` but shouldn't we add the `enterpriseCommit` here when not empty?

Reply:
That's a good idea, this is for tracing / telemetry.

Reply:
Done :)